app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -423,4 +423,8 @@
   <!-- Hints and Solution Fragment -->
   <dimen name="hints_and_solution_recycler_view_padding_top">4dp</dimen>
   <dimen name="hints_and_solution_recycler_view_padding_bottom">112dp</dimen>
+
+  <!-- Options story text size -->
```

Comment:
@shankarpriyank this comment can be written as `Options Story Text Size` or `OptionsStoryTextSize` as this pattern is followed in these files

Reply:
@shankarpriyank Please make this change